src/DynamoPackages/PackageDirectoryBuilder.cs

```diff
@@ -69,24 +69,24 @@ public IDirectoryInfo BuildDirectory(Package package, string packagesDirectory,
         /// </summary>
         /// <param name="package">The package to be formed</param>
         /// <param name="packagesDirectory">The parent directory (the published folder or the default packages directory)</param>
+        /// <param name="roots">All possible root folders for this collection of contentFiles</param>
         /// <param name="contentFiles">The collection of files to be moved</param>
         /// <param name="markdownFiles">Separately provided markdown files</param>
         /// <returns></returns>
-        public IDirectoryInfo BuildRetainDirectory(Package package, string packagesDirectory, IEnumerable<IEnumerable<string>> contentFiles, IEnumerable<string> markdownFiles)
+        public IDirectoryInfo BuildRetainDirectory(Package package, string packagesDirectory, IEnumerable<string> roots, IEnumerable<IEnumerable<string>> contentFiles, IEnumerable<string> markdownFiles)
         {
             
             var rootPath = Path.Combine(packagesDirectory, package.Name);
             var rootDir = fileSystem.TryCreateDirectory(rootPath);
-            var sourcePackageDir = package.RootDirectory;
             package.RootDirectory = rootDir.FullName;
 
             var dyfFiles = new List<string>();
 
             RemoveUnselectedFiles(contentFiles.SelectMany(files => files).ToList(), rootDir);
-            CopyFilesIntoRetainedPackageDirectory(contentFiles, markdownFiles, sourcePackageDir, rootDir, out dyfFiles);
+            CopyFilesIntoRetainedPackageDirectory(contentFiles, markdownFiles, roots, rootDir, out dyfFiles);
             RemoveRetainDyfFiles(contentFiles.SelectMany(files => files).ToList(), dyfFiles);  
```

Comment:
I think this can be removed now?

Reply:
https://github.com/DynamoDS/Dynamo/pull/15374 - I just added a separate PR for that, so we can trace the issue and revert, if need be.